File: teste.rb
def primo(num)
	if num < 2
		return false
	end
	for i in 2..(num - 1)
		if num % i == 0
			return false
		end
	end
	return true
end

for i in 0..(100000 - 1)
	if primo(i)
		puts i
	end
end

File: teste.js
function primo(num) {
	if (num < 2)
		return false;
	for (let i = 2; i < num; i++) {
		if (num % i == 0)
			return false;
	}
	return true;
}

for (let i = 0; i < 100000; i++) {
	if (primo(i))
		console.log(i);
}